src/TodoListItem.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 const TodoListItem = ({ todo, onRemoveTodo}) => {
   return (
-      <li key={todo.id}>{todo.title} <button onClick={()=>onRemoveTodo(todo.id)}>X</button></li>
+      <li key={todo.id}>{todo.fields.Title} <button onClick={()=>onRemoveTodo(todo.id)}>X</button></li>
```

Comment:
don't need to add key here, it is already been added to TodoListItem

Reply:
Hi! do you mean no id in the button?

Reply:
Hi, I mean on 'li' item